kong/conf_loader.lua

```diff
@@ -597,6 +598,34 @@ local function check_and_infer(conf, opts)
   -- custom validations
   ---------------------
 
+  conf.host_ports = {}
+  if conf.port_map then
+    local MIN_PORT = 1
+    local MAX_PORT = 65535
+
+    for _, ports in ipairs(conf.port_map) do
```

Comment:
nit: `ports` is a confusing name. How about `port_map`? See the comment in kong.conf.default on the port_map name as well.

Reply:
changed.